### PR TITLE
Add title, categories, introduction, related work and fix language and typos in fingerprint techniques

### PR DESCRIPTION
**Pull request by Xenonym**
Apart from adding the introduction and related work, I also added:
- title
- author info
- categories
- keywords
and fixed the typos and language weirdness in the fingerprint technology section.

@githubxxcc please help check I didn't mess up the latex! ><

### CODE REVIEW

File: paper.tex [hunk 1]
```diff
@@ -167,79 +167,77 @@ \subsection{Surveys of Fingerprinting Practices}
...
+The rest of this section will go through in more detail about each of them.
 
 \paragraph{JavaScript-Based Fingerprinting}
-JavaScript is the default scripting language for almost all interactive web applications now. It provides functionalities such as rendering of animation, submitting of user forms, and altering the content of the page dynamically. 
-Other than the informative objects mentioned earlier (\verb|navigator| and \verb|screen|), JavaScript can also be used to identify users indirectly, such as obtaining the information of the browsing history, the nuances in performance of JavaScript engine at different browsers.
+JavaScript is the default scripting language for almost all interactive web applications now. It provides functionality such as rendering of animation, submitting of user forms, and altering the content of the page dynamically. 
+Other than the informative objects mentioned earlier (\verb|navigator| and \verb|screen|), JavaScript can also be used to identify users indirectly, such as obtaining  browsing history or fine-grained timing information of the JavaScript engine at different browsers.
```
Comment (KeithSOC) [inline paper.tex]: Do take note of extra space here:

```suggestion
Other than the informative objects mentioned earlier (\verb|navigator| and \verb|screen|), JavaScript can also be used to identify users indirectly, such as obtaining browsing history or fine-grained timing information of the JavaScript engine at different browsers.
```

File: paper.tex [hunk 2]
```diff
@@ -167,79 +167,77 @@ \subsection{Surveys of Fingerprinting Practices}
...
 
 \paragraph{Plugin-Based Fingerprinting}
-Plugins nowadays are equipped with capabilities to access the software and hardware properties on the underlying system. One of these common plugins is Flash, that is used for displaying videos and games in the \verb|.swf| format. In order for the web pages to use Flash effectively, Flash exposes APIs to query for the information on the operating system, as well as the specific version of Flash being installed.
-Moreover, Flash also exposes APIs that allow one to list all the system fonts, which can vary from users to users. The difference in those fonts installed can again be used as an identifier for the user.
-Therefore, the more plugins a user installed, the more information that those plugins expose to fingerprinters.
+Plugins are equipped with capabilities to access the software and hardware properties of the underlying system. One of these common plugins is Flash that is used for displaying videos and games in the \verb|.swf| format. In order for the web pages to use Flash effectively, Flash exposes APIs to query  information about the operating system, as well as the specific version of Flash installed.
```
Comment (KeithSOC) [inline paper.tex]: Do take note of extra space here:

```suggestion
Plugins are equipped with capabilities to access the software and hardware properties of the underlying system. One of these common plugins is Flash that is used for displaying videos and games in the \verb|.swf| format. In order for the web pages to use Flash effectively, Flash exposes APIs to query information about the operating system, as well as the specific version of Flash installed.
```

File: paper.tex [hunk 3]
```diff
@@ -167,79 +167,77 @@ \subsection{Surveys of Fingerprinting Practices}
...
+Moreover, Flash also exposes APIs that allow one to list all the system fonts, which can vary from user to user. The difference in those fonts installed can again be used as an identifier for the user.
+Therefore, the more plugins a user has installed, the more information can be exposed to fingerprinting software.
 
 \paragraph{Extension-Based Fingerprinting}
-A browser extension is a special software component that adds extra functionalities to the browsers, just like how one can download additional applications from an App Store. Common extensions are like RSS feeders, Evernote, etc. They are different from plugins as extensions provide additional functionalities to the browser while plugins enable some functionalities so that web page can be rendered effectively. Take one of the extensions that expose additional
+A browser extension is a special software component that adds extra functionality to the browser, such as RSS reader or ad-blockers. Extensions are different from plugins as extensions provide additional functionality in the browser itself while plugins enable additional functionality for rendering a web page. Take one of the extensions  that expose additional
```
Comment (KeithSOC) [inline paper.tex]: Do take note of extra space here:

```suggestion
A browser extension is a special software component that adds extra functionality to the browser, such as RSS reader or ad-blockers. Extensions are different from plugins as extensions provide additional functionality in the browser itself while plugins enable additional functionality for rendering a web page. Take one of the extensions that expose additional
```

File: paper.tex [hunk 4]
```diff
@@ -167,79 +167,77 @@ \subsection{Surveys of Fingerprinting Practices}
...
+Fingerprint information is used as an additional authentication dimension alongside the normal password-based authentication (or other primary authentication method), and an authenticated session requires the matching of both the primary response as well as the fingerprint data stored.
 In order for this strategy to work, the server must be able to store persistently the fingerprint data which is given by the user at registration. 
-Such strategy can go hand-in-hand with the current two-factor authentification. For example, in the event where the device used for receiving SMS is stolen, fingerprint of the stolen device might reveal the unsual state of it as the fingerprint check failed, which leads to additional security checks.
+Such a strategy can be combined two-factor authentification. For example, in the event where the device used for receiving SMS is stolen, a fingerprint of the stolen device might reveal the unusual state of it as the fingerprint check failed, which leads to additional security checks.
```
Comment (KeithSOC) [inline paper.tex]:
```suggestion
Such a strategy can be combined two-factor authentication. For example, in the event where the device used for receiving SMS is stolen, a fingerprint of the stolen device might reveal the unusual state of it as the fingerprint check failed, which leads to additional security checks.
```

**Review by KeithSOC — CHANGES_REQUESTED** [review]
Great! 👍 Just some minor changes required~

**Comment by rickyyx** [comment]
LGTM. Thanks for the typo and english check. Should have done that by my own.

**Pull request merged**